orte/mca/iof/base/iof_base_output.c

```diff
@@ -320,6 +331,7 @@ void orte_iof_base_write_handler(int fd, short event, void *cbdata)
             return;
         }
         num_written = write(wev->fd, output->data, output->numbytes);
+        cum_written += num_written;
```

Comment:
I guess it doesn't really matter, does it? If num_written is negative, you're just going to return anyway, and so cum_written doesn't matter.

Other than this one point, which is up to you since it should work okay (just looks a little odd), it's fine with me to commit.